css/style.css

```diff
@@ -0,0 +1,384 @@
+@import url('https://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+* {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+}
+
+.menu2 {
+  color: white;
+  padding-right: 5px;
+}
+
+.menu-mobile-wrap {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(9, 9, 9, 0.9);
+  z-index: 1000;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.menu-position {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  cursor: pointer;
+  font-size: 3rem;
+  color: white;
+}
+
+.list-mobile {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.top-menu {
+  display: none;
+}
+
+.top-menu-wrap {
+  display: none;
+}
+
+.custom-toggler {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 24px;
+  height: 3px;
+  background-color: #000;
+  margin-bottom: 4px;
+}
+
+.bg-navbar {
+  height: 130px;
+  padding-bottom: 90px;
+}
+
+.container-fluid {
+  padding: 0%;
+}
+
+.navbar-toggler {
+  border: none;
+}
+
+.bg-header {
+  background: url('../images/backheader2.jpg');
+  width: 100%;
+  height: auto;
+}
+
+.text-link {
+  font-size: 30px;
+}
+
+.text-link-mobile {
+  font-size: 40px;
+  color: white;
+}
+
+.red {
+  color: #ec5242 !important;
+}
+
+.white {
+  color: white;
+}
+
+li {
+  padding-right: 70px;
+}
+
+.b-desktop {
+  display: none;
+}
+
+.b-mobile {
+  display: block;
+}
+
+.padding-0 {
+  padding: 0%;
+}
+
+.logo {
+  width: 100px;
+}
+
+.border-btn {
+  border: 4px solid #ec5242;
+  border-radius: 0%;
+}
+
+.bg-banner {
+  background: url('../images/fondo1.jpg');
+  width: 100%;
+  height: auto;
+}
+
+.banner-subtitle {
+  font-size: 30px;
+  color: #ec5242;
+  margin-top: 150px;
+}
+
+.banner-title {
+  font-family: 'cocogoose', 'Lato', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  color: #ec5242;
+  margin-top: 0%;
+  background: url('../images/backletters5.jpg');
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.banner-wrap-text {
+  font-size: 14px;
+  padding: 19px;
+  background-color: #f3f3f3;
+  border: 2px solid #fff;
+  border-radius: 0%;
+  margin-bottom: 20px;
+}
+
+.banner-wrap-text-about {
+  font-size: 14px;
+  padding: 19px;
+  background-color: #f3f3f3;
+  border: 2px solid #d3d3d3;
+  border-radius: 0%;
+  margin-bottom: 20px;
+}
+
+.banner-text {
+  margin-bottom: 0%;
+}
+
+/* .banner-date {
+} */
+
```

Comment:
Kindly take away every commented block of code as they make your code base look messy👍